Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -19,25 +20,28 @@ class TransactionHistoryViewController: UIViewController {
         return refreshControl
     }()
     
-    @IBOutlet weak var syncLabel: UILabel!
     
+    @IBOutlet weak var syncLabel: UILabel!
     @IBOutlet var tableView: UITableView!
     @IBOutlet var btnFilter: DropMenuButton!
-    var FromMenu = true
-    var visible:Bool = false
     
-    var filterMenu = [LocalizedStrings.all] as [String]
-    var filtertitle = [0] as [Int]
+    var filterMenu = [LocalizedStrings.all]
+    var filters: [Int32] = [0]
     
-    var mainContens = [Transaction]()
-    var Filtercontent = [Transaction]()
+    var transactions = [Transaction]()
+    var filteredItems = [Transaction]() {
```

Comment:
This is a nice improvement from the previous name `transactions`. I do think `filteredTxs/filteredTransactions/displayedTransactions/displayedTxs` would make better choices.
Also, `transactions` could become `allTransactions` or `allTxs`.

Reply:
fixed in 9a8c24b